Source/interfaces/IDolby.h

```diff
@@ -38,9 +38,50 @@ namespace Exchange {
                 DIGITAL_AC3 = 4,
                 AUTO = 5
             };
-            
+
+            enum SoundModes : uint8_t {
```

Comment:
I have the impression (but you need to confirm it on ComCast side) That the enum type (just above here)  and SoundModes are the same, just different (or more values) If so, I think we can than combine

Reply:
RDK actually maintains them as separate enums.

The Type enum here is analogous to our "audio encoding" enum,

```
/**
 * This enumeration defines the type of audio encoding that can be used.
 */
typedef enum _dsAudioEncoding_t{
    dsAUDIO_ENC_NONE = 0,        /**< No digital audio output.                         */
    dsAUDIO_ENC_DISPLAY,     /**< Platform-selected digital audio encoding format. */
    dsAUDIO_ENC_PCM,         /**< PCM digital audio encoding format.               */
    dsAUDIO_ENC_AC3,         /**< AC-3 digital audio encoding format.               */
    dsAUDIO_ENC_EAC3,        /**< DD+/E-AC-3 digital audio encoding format.               */
    dsAUDIO_ENC_MAX          /**< Maximum index for audio encoding types.          */
} dsAudioEncoding_t;
```

Whereas SoundMode is maintained separately
```
/**
 * This enumeration defines the type of audio stereo mode that can be used.
 */
typedef enum StereoMode{
    dsAUDIO_STEREO_UNKNOWN,   /**< Unknown mode.                         */
    dsAUDIO_STEREO_MONO = 1,  /**< Mono mode.                            */
    dsAUDIO_STEREO_STEREO,    /**< Normal stereo mode (L+R).             */
    dsAUDIO_STEREO_SURROUND,  /**< Surround mode.                        */
    dsAUDIO_STEREO_PASSTHRU,  /**< Passthrough mode.                     */
    dsAUDIO_STEREO_MAX        /**< Maximum index for audio stereo types. */
} dsAudioStereoMode_t;

```
I believe it is best to retain Types and SoundModes as separate

Reply:
Oke